pallets/collator-selection/src/benchmarking.rs

```diff
@@ -134,21 +134,21 @@ benchmarks! {
 
 	// worse case is paying a non-existing candidate account.
 	note_author {
-		let c in 1 .. T::MaxCandidates::get();
 		<CandidacyBond<T>>::put(T::Currency::minimum_balance());
-		<DesiredCandidates<T>>::put(c);
-		register_candidates::<T>(c);
-
 		T::Currency::make_free_balance_be(
 			&<CollatorSelection<T>>::account_id(),
-			T::Currency::minimum_balance() * 2u32.into(),
+			T::Currency::minimum_balance() * 4u32.into(),
 		);
 		let author = account("author", 0, SEED);
+		let new_block: T::BlockNumber = 10u32.into();
+
+		frame_system::Pallet::<T>::set_block_number(new_block.clone());
```

Comment:
```suggestion
		frame_system::Pallet::<T>::set_block_number(new_block);
```

isn't block number `Copy`?